apps/explorer/cypress/e2e/static/e2e.cy.ts

```diff
@@ -97,22 +97,6 @@ describe('End-to-end Tests', () => {
             cy.get('#objectID').contains('CollectionObject');
         });
 
-        it('where last and next disappear in final page', () => {
```

Comment:
I think there's a bunch of other tests that should be removed right?

Reply:
I removed all the e2e tests. Most of this is done in the unit test